modules/luci-mod-dashboard/htdocs/luci-static/resources/view/dashboard/include/20_lan.js

```diff
@@ -45,10 +35,18 @@ return baseclass.extend({
 			E('h3', this.title)
 		]));
 
+		var container_devices = E('table', { 'class': 'table assoclist devices-info' }, [
+			E('tr', { 'class': 'tr dashboard-bg' }, [
+				E('th', { 'class': 'th nowrap' }, _('Hostname')),
+				E('th', { 'class': 'th' }, _('IP Address')),
+				E('th', { 'class': 'th' }, _('MAC')),
+			])
+		]);
+
 		for(var idx in this.params.lan.devices) {
 			var device = this.params.lan.devices[idx];
 
-			container_deviceslist.appendChild(E('tr', { 'class': 'tr cbi-rowstyle-1'}, [
+			container_devices.appendChild(E('tr', { 'class': 'tr cbi-rowstyle-1'}, [
```

Comment:
Perhaps some alternate row styling based on idx being odd/even (use modulo).
```js
			container_devices.appendChild(E('tr', { 'class': (idx % 2) ? 'tr cbi-rowstyle-2' : 'tr cbi-rowstyle-1'}, [
```

Reply:
For now I didn't really try to fix the CSS logic, just the DOM logic.